modules/custom/dkan_api/src/Controller/Organization.php

```diff
@@ -2,10 +2,15 @@
 
 namespace Drupal\dkan_api\Controller;
 
-class Organization extends Api
-{
-  protected function getJsonSchema()
-  {
+/**
+ *
+ */
+class Organization extends Api {
+
+  /**
+   *
+   */
+  protected function getJsonSchema() {
     "
```

Comment:
missing `return` ? calling the method returns nothing.

Reply:
@ketwaroo This class is not being used right now, and it might be removed. Lets not test it, or refactor it.

Reply:
okay. I'll just leave it as is until such time you decide to remove it. the incomplete coverage will be like a reminder that it's an outstanding issue.